modules/@apostrophecms/ui/ui/apos/components/AposSlatList.vue

```diff
@@ -32,6 +32,7 @@
           :slat-count="next.length"
           :removable="removable"
           :has-relationship-schema="hasRelationshipSchema"
+          :image-editor="imageEditor"
```

Comment:
Again should be the configured component name, should not assume it has to be images.